app/main.py

```diff
@@ -1,4 +1,8 @@
 def get_human_age(cat_age: int, dog_age: int) -> list:
+    if cat_age <= 0 or dog_age <= 0:
+        raise ValueError
+    if cat_age >= 100 or dog_age >= 100:
+        raise ValueError
```

Comment:
You should not modify the main function, work with it in its initial state. Exclude this file from the pull request.